### ISSUE TITLE
conda-forge Linux CI  Failure Begin of November 2022 due to Python 3.11 update

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#1298]
~~~
2022-11-03T02:53:10.4233446Z [149/424] Performing build step for 'manif'
2022-11-03T02:53:10.4233955Z FAILED: src/manif/CMakeFiles/YCMStamp/manif-build /home/runner/work/robotology-superbuild/robotology-superbuild/build/src/manif/CMakeFiles/YCMStamp/manif-build 
2022-11-03T02:53:10.4234703Z cd /home/runner/work/robotology-superbuild/robotology-superbuild/build/src/manif && /usr/share/miniconda3/envs/test/bin/cmake --build . && /usr/share/miniconda3/envs/test/bin/cmake -E touch /home/runner/work/robotology-superbuild/robotology-superbuild/build/src/manif/CMakeFiles/YCMStamp/manif-build
2022-11-03T02:53:10.4235215Z [1/8] Building CXX object python/CMakeFiles/manifpy.dir/bindings_rn.cpp.o
2022-11-03T02:53:10.4235466Z FAILED: python/CMakeFiles/manifpy.dir/bindings_rn.cpp.o 
2022-11-03T02:53:10.4236939Z /usr/share/miniconda3/envs/test/bin/x86_64-conda-linux-gnu-c++ -DEIGEN_DEFAULT_TO_ROW_MAJOR -Dmanifpy_EXPORTS -I/home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/include -I/home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/external/tl -isystem /usr/include/python3.8 -isystem /usr/share/miniconda3/envs/test/include/eigen3 -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /usr/share/miniconda3/envs/test/include -O3 -DNDEBUG -fPIC -fvisibility=hidden -flto -fno-fat-lto-objects -std=c++11 -MD -MT python/CMakeFiles/manifpy.dir/bindings_rn.cpp.o -MF python/CMakeFiles/manifpy.dir/bindings_rn.cpp.o.d -o python/CMakeFiles/manifpy.dir/bindings_rn.cpp.o -c /home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/python/bindings_rn.cpp
2022-11-03T02:53:10.4237938Z In file included from /usr/include/python3.8/Python.h:8,
2022-11-03T02:53:10.4238184Z                  from /usr/share/miniconda3/envs/test/include/pybind11/detail/common.h:208,
2022-11-03T02:53:10.4238431Z                  from /usr/share/miniconda3/envs/test/include/pybind11/attr.h:13,
2022-11-03T02:53:10.4238676Z                  from /usr/share/miniconda3/envs/test/include/pybind11/detail/class.h:12,
2022-11-03T02:53:10.4239083Z                  from /usr/share/miniconda3/envs/test/include/pybind11/pybind11.h:13,
2022-11-03T02:53:10.4239462Z                  from /home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/python/bindings_rn.cpp:1:
2022-11-03T02:53:10.4239877Z /usr/include/python3.8/pyconfig.h:3:12: fatal error: x86_64-linux-gnu/python3.8/pyconfig.h: No such file or directory
2022-11-03T02:53:10.4240200Z     3 | #  include <x86_64-linux-gnu/python3.8/pyconfig.h>
2022-11-03T02:53:10.4240391Z       |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2022-11-03T02:53:10.4240547Z compilation terminated.
2022-11-03T02:53:10.4240762Z [2/8] Building CXX object python/CMakeFiles/manifpy.dir/bindings_so2.cpp.o
2022-11-03T02:53:10.4241018Z FAILED: python/CMakeFiles/manifpy.dir/bindings_so2.cpp.o 
2022-11-03T02:53:10.4242472Z /usr/share/miniconda3/envs/test/bin/x86_64-conda-linux-gnu-c++ -DEIGEN_DEFAULT_TO_ROW_MAJOR -Dmanifpy_EXPORTS -I/home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/include -I/home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/external/tl -isystem /usr/include/python3.8 -isystem /usr/share/miniconda3/envs/test/include/eigen3 -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /usr/share/miniconda3/envs/test/include -O3 -DNDEBUG -fPIC -fvisibility=hidden -flto -fno-fat-lto-objects -std=c++11 -MD -MT python/CMakeFiles/manifpy.dir/bindings_so2.cpp.o -MF python/CMakeFiles/manifpy.dir/bindings_so2.cpp.o.d -o python/CMakeFiles/manifpy.dir/bindings_so2.cpp.o -c /home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/python/bindings_so2.cpp
2022-11-03T02:53:10.4243502Z In file included from /usr/include/python3.8/Python.h:8,
2022-11-03T02:53:10.4243737Z                  from /usr/share/miniconda3/envs/test/include/pybind11/detail/common.h:208,
2022-11-03T02:53:10.4243987Z                  from /usr/share/miniconda3/envs/test/include/pybind11/attr.h:13,
2022-11-03T02:53:10.4244236Z                  from /usr/share/miniconda3/envs/test/include/pybind11/detail/class.h:12,
2022-11-03T02:53:10.4244491Z                  from /usr/share/miniconda3/envs/test/include/pybind11/pybind11.h:13,
2022-11-03T02:53:10.4244861Z                  from /home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/python/bindings_so2.cpp:1:
2022-11-03T02:53:10.4245267Z /usr/include/python3.8/pyconfig.h:3:12: fatal error: x86_64-linux-gnu/python3.8/pyconfig.h: No such file or directory
2022-11-03T02:53:10.4245579Z     3 | #  include <x86_64-linux-gnu/python3.8/pyconfig.h>
2022-11-03T02:53:10.4245767Z       |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2022-11-03T02:53:10.4245922Z compilation terminated.
2022-11-03T02:53:10.4246137Z [3/8] Building CXX object python/CMakeFiles/manifpy.dir/bindings_so3.cpp.o
2022-11-03T02:53:10.4246376Z FAILED: python/CMakeFiles/manifpy.dir/bindings_so3.cpp.o 
2022-11-03T02:53:10.4247777Z /usr/share/miniconda3/envs/test/bin/x86_64-conda-linux-gnu-c++ -DEIGEN_DEFAULT_TO_ROW_MAJOR -Dmanifpy_EXPORTS -I/home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/include -I/home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/external/tl -isystem /usr/include/python3.8 -isystem /usr/share/miniconda3/envs/test/include/eigen3 -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /usr/share/miniconda3/envs/test/include -O3 -DNDEBUG -fPIC -fvisibility=hidden -flto -fno-fat-lto-objects -std=c++11 -MD -MT python/CMakeFiles/manifpy.dir/bindings_so3.cpp.o -MF python/CMakeFiles/manifpy.dir/bindings_so3.cpp.o.d -o python/CMakeFiles/manifpy.dir/bindings_so3.cpp.o -c /home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/python/bindings_so3.cpp
2022-11-03T02:53:10.4248726Z In file included from /usr/include/python3.8/Python.h:8,
2022-11-03T02:53:10.4248992Z                  from /usr/share/miniconda3/envs/test/include/pybind11/detail/common.h:208,
2022-11-03T02:53:10.4249249Z                  from /usr/share/miniconda3/envs/test/include/pybind11/attr.h:13,
2022-11-03T02:53:10.4249499Z                  from /usr/share/miniconda3/envs/test/include/pybind11/detail/class.h:12,
2022-11-03T02:53:10.4249745Z                  from /usr/share/miniconda3/envs/test/include/pybind11/pybind11.h:13,
2022-11-03T02:53:10.4250115Z                  from /home/runner/work/robotology-superbuild/robotology-superbuild/src/manif/python/bindings_so3.cpp:1:
2022-11-03T02:53:10.4250515Z /usr/include/python3.8/pyconfig.h:3:12: fatal error: x86_64-linux-gnu/python3.8/pyconfig.h: No such file or directory
2022-11-03T02:53:10.4250826Z     3 | #  include <x86_64-linux-gnu/python3.8/pyconfig.h>
2022-11-03T02:53:10.4251005Z       |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2022-11-03T02:53:10.4251164Z compilation terminated.
2022-11-03T02:53:10.4251342Z ninja: build stopped: subcommand failed.
~~~

**Comment by traversaro** [comment]
It seems that for some reason manifpy for some reason is finding the system python instead of the conda one.

**Comment by traversaro** [comment]
I think it is a pybind11 regression. manif does not directly search for Python, but just for pybind11 that itself searches for python, see https://github.com/artivis/manif/blob/805a0b2adf9435dd658fdad5606c342db4d889e8/CMakeLists.txt#L64 .

**Comment by traversaro** [comment]
I did a diff between the last job that was building manif correctly and the first that failed:
~~~patch
--- <unnamed>
+++ <unnamed>
@@ -2,27 +2,31 @@
 _openmp_mutex             4.5                       2_gnu    conda-forge
 _sysroot_linux-64_curr_repodata_hack 3                   h5bd9786_13    conda-forge
 ace                       7.0.9                h27087fc_0    conda-forge
-alsa-lib                  1.2.7.2              h166bdaf_0    conda-forge
+aiohttp                   3.8.3           py311hd4cff14_1    conda-forge
+aiosignal                 1.2.0              pyhd8ed1ab_0    conda-forge
+alsa-lib                  1.2.8                h166bdaf_0    conda-forge
 ampl-mp                   3.1.0             h2cc385e_1006    conda-forge
 aom                       3.5.0                h27087fc_0    conda-forge
 asio                      1.24.0               h27087fc_0    conda-forge
 assimp                    5.2.5                hf40c2ba_0    conda-forge
 asttokens                 2.0.8              pyhd8ed1ab_0    conda-forge
+async-timeout             4.0.2              pyhd8ed1ab_0    conda-forge
 atk-1.0                   2.36.0               h3371d22_4    conda-forge
 attr                      2.5.1                h166bdaf_1    conda-forge
+attrs                     22.1.0             pyh71513ae_1    conda-forge
 backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
 backports                 1.0                        py_2    conda-forge
 backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
-bash-completion           2.11                          0    conda-forge
+bash-completion           2.11                 ha770c72_1    conda-forge
 binutils                  2.39                 hdd6e379_0    conda-forge
 binutils_impl_linux-64    2.39                 h6ceecb4_0    conda-forge
 binutils_linux-64         2.39                h5fc0e48_11    conda-forge
 blosc                     1.21.1               h83bc5f7_3    conda-forge
-boost                     1.78.0          py310hc4a4660_4    conda-forge
+boost                     1.78.0          py311h59ea3da_4    conda-forge
 boost-cpp                 1.78.0               h75c5d50_1    conda-forge
 brotli                    1.0.9                h166bdaf_8    conda-forge
 brotli-bin                1.0.9                h166bdaf_8    conda-forge
-bullet-cpp                3.21                 h769672d_4    conda-forge
+bullet-cpp                3.21                 h8b32b4d_4    conda-forge
 bzip2                     1.0.8                h7f98852_4    conda-forge
 c-ares                    1.18.1               h7f98852_0    conda-forge
 c-compiler                1.5.1                h166bdaf_0    conda-forge
@@ -32,11 +36,12 @@
 cairo                     1.16.0            ha61ee94_1014    conda-forge
 certifi                   2022.9.24          pyhd8ed1ab_0    conda-forge
 cfitsio                   4.1.0                hd9d235c_0    conda-forge
+charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
 cli11                     2.3.0                h27087fc_0    conda-forge
 cmake                     3.24.2               h5432695_0    conda-forge
 compilers                 1.5.1                ha770c72_0    conda-forge
 console_bridge            1.0.2                h924138e_1    conda-forge
-contourpy                 1.0.5           py310hbf28c38_1    conda-forge
+contourpy                 1.0.6           py311h4dd048b_0    conda-forge
 cppzmq                    4.8.1                hf7cf922_0    conda-forge
 curl                      7.86.0               h7bff187_0    conda-forge
 cxx-compiler              1.5.1                h924138e_0    conda-forge
@@ -50,7 +55,7 @@
 expat                     2.5.0                h27087fc_0    conda-forge
 expat-cos7-x86_64         2.1.0             h9b0a68f_1105    conda-forge
 fcl                       0.7.0                hdebe893_2    conda-forge
-ffmpeg                    4.4.2           gpl_hc51e5dc_110    conda-forge
+ffmpeg                    5.1.2           gpl_hc51e5dc_103    conda-forge
 fftw                      3.3.10          nompi_hf0379b8_105    conda-forge
 flann                     1.9.1             he05ef13_1011    conda-forge
 font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
@@ -60,14 +65,15 @@
 fontconfig                2.14.1               hc2a2eb6_0    conda-forge
 fonts-conda-ecosystem     1                             0    conda-forge
 fonts-conda-forge         1                             0    conda-forge
-fonttools                 4.38.0          py310h5764c6d_1    conda-forge
+fonttools                 4.38.0          py311hd4cff14_1    conda-forge
 fortran-compiler          1.5.1                h2a4ca65_0    conda-forge
 freeglut                  3.2.2                h9c3ff4c_1    conda-forge
-freeimage                 3.18.0               he1b5a44_0    conda-forge
+freeimage                 3.18.0              h530e30e_10    conda-forge
 freetype                  2.12.1               hca18f0e_0    conda-forge
 freexl                    1.0.6                h166bdaf_1    conda-forge
 fribidi                   1.0.10               h36c2ea0_0    conda-forge
-gazebo                    11.12.0              h06ed1b4_2    conda-forge
+frozenlist                1.3.1           py311hd4cff14_1    conda-forge
+gazebo                    11.12.0              hab95640_3    conda-forge
 gcc                       10.4.0              hb92f740_11    conda-forge
 gcc_impl_linux-64         10.4.0              h5231bdf_19    conda-forge
 gcc_linux-64              10.4.0              h9215b83_11    conda-forge
@@ -91,35 +97,37 @@
 graphite2                 1.3.13            h58526e2_1001    conda-forge
 graphviz                  6.0.1                h5abf519_0    conda-forge
 gsl                       2.7                  he838d99_0    conda-forge
-gst-plugins-base          1.20.3               h57caac4_2    conda-forge
-gstreamer                 1.20.3               hd4edc92_2    conda-forge
+gst-plugins-base          1.21.1               h3e40eee_1    conda-forge
+gstreamer                 1.21.1               hd4edc92_1    conda-forge
 gtk2                      2.24.33              h90689f9_2    conda-forge
 gts                       0.7.6                h64030ff_2    conda-forge
 gxx                       10.4.0              hb92f740_11    conda-forge
 gxx_impl_linux-64         10.4.0              h5231bdf_19    conda-forge
 gxx_linux-64              10.4.0              h6e491c6_11    conda-forge
-h5py                      3.7.0           nompi_py310h416281c_102    conda-forge
+h5py                      3.7.0           nompi_py311hbe7f6d8_102    conda-forge
 harfbuzz                  5.3.0                h418a68e_0    conda-forge
-hdf4                      4.2.15               h9772cbc_4    conda-forge
+hdf4                      4.2.15               h9772cbc_5    conda-forge
 hdf5                      1.12.2          nompi_h2386368_100    conda-forge
 icu                       70.1                 h27087fc_0    conda-forge
+idna                      3.4                pyhd8ed1ab_0    conda-forge
 imath                     3.1.5                h6239696_0    conda-forge
 ipopt                     3.14.10              hf60ba86_0    conda-forge
 ipython                   8.6.0              pyh41d4057_1    conda-forge
 irrlicht                  1.8.5                h80a045a_2    conda-forge
-jack                      1.9.21               h2a1e645_0    conda-forge
+jack                      1.9.21               he978b8e_1    conda-forge
 jasper                    2.0.33               ha77e612_0    conda-forge
 jedi                      0.18.1             pyhd8ed1ab_2    conda-forge
 jpeg                      9e                   h166bdaf_2    conda-forge
 json-c                    0.16                 hc379101_0    conda-forge
 jsoncpp                   1.9.5                h4bd325d_1    conda-forge
+jxrlib                    1.1                  h7f98852_2    conda-forge
 kealib                    1.4.15               ha7026e8_1    conda-forge
 kernel-headers_linux-64   3.10.0              h4a8ded7_13    conda-forge
 keyutils                  1.6.1                h166bdaf_0    conda-forge
-kiwisolver                1.4.4           py310hbf28c38_1    conda-forge
+kiwisolver                1.4.4           py311h4dd048b_1    conda-forge
 krb5                      1.19.3               h3790be6_0    conda-forge
 lame                      3.100             h166bdaf_1003    conda-forge
-lcms2                     2.13.1               h6ed2654_0    conda-forge
+lcms2                     2.12                 hddcbb42_0    conda-forge
 ld_impl_linux-64          2.39                 hc81fddc_0    conda-forge
 lerc                      4.0.0                h27087fc_0    conda-forge
 libblas                   3.9.0           16_linux64_openblas    conda-forge
@@ -129,8 +137,8 @@
 libcap                    2.66                 ha37c62d_0    conda-forge
 libcblas                  3.9.0           16_linux64_openblas    conda-forge
 libccd-double             2.1                  h27087fc_3    conda-forge
-libclang                  14.0.6          default_h2e3cab8_0    conda-forge
-libclang13                14.0.6          default_h3a83d3e_0    conda-forge
+libclang                  15.0.3          default_h2e3cab8_0    conda-forge
+libclang13                15.0.3          default_h3a83d3e_0    conda-forge
 libcups                   2.3.3                h3e49a29_2    conda-forge
 libcurl                   7.86.0               h7bff187_0    conda-forge
 libdap4                   3.20.6               hd7c4107_2    conda-forge
@@ -147,7 +155,7 @@
 libgcc-devel_linux-64     10.4.0              hd38fd1e_19    conda-forge
 libgcc-ng                 12.2.0              h65d4601_19    conda-forge
 libgd                     2.3.3                h18fbbfe_3    conda-forge
-libgdal                   3.5.2                hf663712_5    conda-forge
+libgdal                   3.5.2                hf663712_6    conda-forge
 libgfortran-ng            12.2.0              h69a702a_19    conda-forge
 libgfortran5              12.2.0              h337968e_19    conda-forge
 libglib                   2.74.1               h7a41b64_0    conda-forge
@@ -159,13 +167,13 @@
 libiconv                  1.17                 h166bdaf_0    conda-forge
 libidn2                   2.3.4                h166bdaf_0    conda-forge
 libignition-cmake2        2.15.0               h27087fc_1    conda-forge
-libignition-common3       3.13.2               h8419eb0_3    conda-forge
-libignition-common4       4.5.1                h8419eb0_1    conda-forge
+libignition-common3       3.15.1               h24a332d_1    conda-forge
+libignition-common4       4.6.1                h24a332d_0    conda-forge
 libignition-fuel-tools4   4.6.0                h08d22d4_0    conda-forge
 libignition-fuel-tools7   7.1.0                h08d22d4_1    conda-forge
 libignition-gazebo6       6.12.0               h8739608_1    conda-forge
 libignition-gui6          6.6.1                ha101cf2_1    conda-forge
-libignition-math6         6.13.0          py310h3f2b827_1    conda-forge
+libignition-math6         6.13.0          py311hc6c49d4_1    conda-forge
 libignition-msgs5         5.9.0                h07fcb4f_1    conda-forge
 libignition-msgs8         8.6.0                h07fcb4f_1    conda-forge
 libignition-physics5      5.2.0                h27087fc_0    conda-forge
@@ -180,20 +188,21 @@
 libkml                    1.3.0             h37653c0_1015    conda-forge
 liblapack                 3.9.0           16_linux64_openblas    conda-forge
 liblapacke                3.9.0           16_linux64_openblas    conda-forge
-libllvm14                 14.0.6               he0ac6c6_0    conda-forge
+libllvm15                 15.0.3               h63197d8_1    conda-forge
 libmatio                  1.5.23               h63e3022_1    conda-forge
-libnetcdf                 4.8.1           nompi_h261ec11_105    conda-forge
+libnetcdf                 4.8.1           nompi_h261ec11_106    conda-forge
 libnghttp2                1.47.0               hdcd2b5c_1    conda-forge
 libnsl                    2.0.0                h7f98852_0    conda-forge
-libode                    0.16.2              hd8f1fbe_11    conda-forge
+libode                    0.16.2              ha362b79_11    conda-forge
 libogg                    1.3.4                h7f98852_1    conda-forge
 libopenblas               0.3.21          pthreads_h78a6416_3    conda-forge
-libopencv                 4.6.0           py310hd0265ad_4    conda-forge
+libopencv                 4.6.0           py311h8479920_6    conda-forge
 libopus                   1.3.1                h7f98852_1    conda-forge
 libpciaccess              0.16                 h516909a_0    conda-forge
 libpng                    1.6.38               h753d276_0    conda-forge
 libpq                     14.5                 hd77ab85_1    conda-forge
 libprotobuf               3.21.9               h6239696_0    conda-forge
+libraw                    0.20.2               h10796ff_1    conda-forge
 librsvg                   2.54.4               h7abd40a_0    conda-forge
 librttopo                 1.1.0               hf730bdb_11    conda-forge
 libsanitizer              10.4.0              h5246dfb_19    conda-forge
@@ -238,12 +247,12 @@
 libxxf86vm-cos7-x86_64    1.1.4             h9b0a68f_1105    conda-forge
 libzip                    1.9.2                hc869a4a_1    conda-forge
 libzlib                   1.2.13               h166bdaf_4    conda-forge
-loguru                    0.6.0           py310hff52083_2    conda-forge
+loguru                    0.6.0           py311h38be061_2    conda-forge
 lua                       5.4.4                h25fbb91_0    conda-forge
 lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
 make                      4.3                  hd18ef5c_1    conda-forge
-matplotlib                3.6.1           py310hff52083_1    conda-forge
-matplotlib-base           3.6.1           py310h8d5ebf3_1    conda-forge
+matplotlib                3.6.1           py311h38be061_1    conda-forge
+matplotlib-base           3.6.1           py311he728205_1    conda-forge
 matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
 mesa-khr-devel-cos7-x86_64 18.3.4            h9b0a68f_1105    conda-forge
 mesa-libgl-cos7-x86_64    18.3.4            h9b0a68f_1105    conda-forge
@@ -251,6 +260,7 @@
 mesa-libglapi-cos7-x86_64 18.3.4            h9b0a68f_1105    conda-forge
 metis                     5.1.0             h58526e2_1006    conda-forge
 mpg123                    1.30.2               h27087fc_1    conda-forge
+multidict                 6.0.2           py311hd4cff14_2    conda-forge
 mumps-include             5.2.1               ha770c72_11    conda-forge
 mumps-seq                 5.2.1               h2104b81_11    conda-forge
 munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
@@ -262,12 +272,12 @@
 nlohmann_json             3.11.2               h27087fc_0    conda-forge
 nspr                      4.32                 h9c3ff4c_1    conda-forge
 nss                       3.78                 h2350873_0    conda-forge
-numpy                     1.23.4          py310h53a5b5f_1    conda-forge
+numpy                     1.23.4          py311h7d28db0_1    conda-forge
 octomap                   1.9.7                h4bd325d_0    conda-forge
 ogre                      1.10.12             h7cc4a1d_10    conda-forge
 ogre-next                 2.2.6                hd9d6a18_0    conda-forge
 openal-soft               1.22.2               h924138e_0    conda-forge
-opencv                    4.6.0           py310hff52083_4    conda-forge
+opencv                    4.6.0           py311h38be061_6    conda-forge
 openexr                   3.1.5                he0ac6c6_0    conda-forge
 openh264                  2.3.1                h27087fc_1    conda-forge
 openjpeg                  2.5.0                h7d73246_1    conda-forge
@@ -281,36 +291,36 @@
 pcre2                     10.37                hc3806b6_1    conda-forge
 pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
 pickleshare               0.7.5                   py_1003    conda-forge
-pillow                    9.2.0           py310h454ad03_3    conda-forge
+pillow                    9.2.0           py311h9461556_3    conda-forge
 pip                       22.3               pyhd8ed1ab_0    conda-forge
 pixman                    0.40.0               h36c2ea0_0    conda-forge
 pkg-config                0.29.2            h36c2ea0_1008    conda-forge
 ply                       3.11                       py_1    conda-forge
 poppler                   22.10.0              h92391eb_0    conda-forge
 poppler-data              0.4.11               hd8ed1ab_0    conda-forge
-portaudio                 19.6.0               h8e90077_6    conda-forge
+portaudio                 19.6.0               h583fa2b_7    conda-forge
 postgresql                14.5                 hdeef612_1    conda-forge
 proj                      9.1.0                h93bde94_0    conda-forge
 prompt-toolkit            3.0.31             pyha770c72_0    conda-forge
 pthread-stubs             0.4               h36c2ea0_1001    conda-forge
 ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
 pugixml                   1.11.4               h9c3ff4c_0    conda-forge
-pulseaudio                14.0                habe0971_10    conda-forge
+pulseaudio                14.0                h0d2025b_11    conda-forge
 pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
-py-opencv                 4.6.0           py310hfdc917e_4    conda-forge
-pybind11                  2.10.0          py310hbf28c38_1    conda-forge
+py-opencv                 4.6.0           py311h781c19f_6    conda-forge
+pybind11                  2.10.1          py311h4dd048b_0    conda-forge
 pybind11-abi              4                    hd8ed1ab_3    conda-forge
-pybind11-global           2.10.0          py310hbf28c38_1    conda-forge
+pybind11-global           2.10.1          py311h4dd048b_0    conda-forge
 pygments                  2.13.0             pyhd8ed1ab_0    conda-forge
 pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
-pyqt                      5.15.7          py310h29803b5_2    conda-forge
-pyqt5-sip                 12.11.0         py310hd8f1fbe_2    conda-forge
-python                    3.10.6          h582c2e5_0_cpython    conda-forge
+pyqt                      5.15.7          py311h3408d8f_2    conda-forge
+pyqt5-sip                 12.11.0         py311ha362b79_2    conda-forge
+python                    3.11.0          h582c2e5_0_cpython    conda-forge
 python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
-python_abi                3.10                    2_cp310    conda-forge
-pyzmq                     24.0.1          py310h330234f_1    conda-forge
+python_abi                3.11                    2_cp311    conda-forge
+pyzmq                     24.0.1          py311ha4b6469_1    conda-forge
 qhull                     2020.2               h4bd325d_2    conda-forge
-qt-main                   5.15.6               hc525480_0    conda-forge
+qt-main                   5.15.6               hd477bba_1    conda-forge
 qwt                       6.2.0                h1d9fb53_4    conda-forge
 readline                  8.1.2                h0f457ee_0    conda-forge
 rhash                     1.4.3                h166bdaf_0    conda-forge
@@ -320,7 +330,7 @@
 sdl2                      2.0.22               h55ee7ae_2    conda-forge
 setuptools                65.5.0             pyhd8ed1ab_0    conda-forge
 simbody                   3.7                  h64f3f5a_3    conda-forge
-sip                       6.7.3           py310hd8f1fbe_0    conda-forge
+sip                       6.7.3           py311ha362b79_0    conda-forge
 six                       1.16.0             pyh6c4a22f_0    conda-forge
 snappy                    1.1.9                hbd366e4_1    conda-forge
 soxr                      0.1.3                h166bdaf_2    conda-forge
@@ -337,19 +347,21 @@
 tinyxml2                  9.0.0                h9c3ff4c_2    conda-forge
 tk                        8.6.12               h27826a3_0    conda-forge
 toml                      0.10.2             pyhd8ed1ab_0    conda-forge
-tornado                   6.2             py310h5764c6d_1    conda-forge
+tornado                   6.2             py311hd4cff14_1    conda-forge
 traitlets                 5.5.0              pyhd8ed1ab_0    conda-forge
+typing-extensions         4.4.0                hd8ed1ab_0    conda-forge
+typing_extensions         4.4.0              pyha770c72_0    conda-forge
 tzcode                    2022e                h166bdaf_0    conda-forge
 tzdata                    2022e                h191b570_0    conda-forge
 u-msgpack-python          2.7.1              pyh9f0ad1d_0    conda-forge
-unicodedata2              14.0.0          py310h5764c6d_2    conda-forge
 unixodbc                  2.3.10               h583eb01_0    conda-forge
 urdfdom                   3.1.0                h068d047_1    conda-forge
 urdfdom_headers           1.1.0                h924138e_0    conda-forge
 utfcpp                    3.2.1                ha770c72_0    conda-forge
-vtk                       9.2.2           qt_py310haf2e316_201    conda-forge
+vtk                       9.2.2           qt_py311h4f2dc85_204    conda-forge
 wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
 wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
+wslink                    1.9.0              pyhd8ed1ab_0    conda-forge
 x264                      1!164.3095           h166bdaf_2    conda-forge
 x265                      3.5                  h924138e_3    conda-forge
 xcb-util                  0.4.0                h166bdaf_0    conda-forge
@@ -383,6 +395,7 @@
 xorg-xproto               7.0.31            h7f98852_1007    conda-forge
 xz                        5.2.6                h166bdaf_0    conda-forge
 yaml                      0.2.5                h7f98852_2    conda-forge
+yarl                      1.8.1           py311hd4cff14_0    conda-forge
 zeromq                    4.3.4                h9c3ff4c_1    conda-forge
 zlib                      1.2.13               h166bdaf_4    conda-forge
 zstd                      1.5.2                h6239696_4    conda-forge
~~~

**Comment by traversaro** [comment]
This is not a CMake not updated issue as CMake was updated for Python 3.11 a long time ago in https://gitlab.kitware.com/cmake/cmake/-/merge_requests/6792/diffs .

While I investigate, I will pin python to 3.10 .

**Comment by traversaro** [comment]
I found the problem: the fork of manif that we are using (https://github.com/robotology-dependencies/manif/tree/0.0.4.2/cmake) has a vendored version of FindPython that was not updated to support Python 3.11. So the thing to do are:
* 1. Fix the fork and do a 0.0.4.3 release compatible with Python 3.11
* 2. Check if master of manif is working fine and contains all the fix that we need
* 3. If 2. has a positive outcome, kindly ask artivis for an upstream tag and switch to that.
